src/components/Provider.tsx

```diff
@@ -34,12 +34,36 @@ interface IStyedThemeColorOptions {
   themeColors: ThemeColors;
 }
 
-const SName = styled.div<IStyedThemeColorOptions>`
+const SName = styled.button<IStyedThemeColorOptions>`
```

Comment:
We need to make one of card's elements really interactive to grant the ability to use it as focusable element.